.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Create Release
+    about: Please create the release issue in the internal Issue Tracker repository. - _Only accessible by maintainers._
+    url: https://github.com/elastisys/ck8s-issue-tracker/issues/new?template=release-apps.md
```

Comment:
What is the intention behind moving the release and patch issues to the internal tracker? I'm asking because we really like to show the QA checklist publicly: https://elastisys.io/welkin/quality-criteria/#quality-assurance-qa